guides/common/modules/proc_salt-deploying-minion-hosts.adoc

```diff
@@ -0,0 +1,17 @@
+[id="salt_guide_deploying_minion_hosts_{context}"]
+= Deploying Minion Hosts
+
+Deploy managed hosts that are fully provisioned and configured for Salt usage.
+
+.Prerequisites
+. You have set up a _Salt Master_ and _Salt Environment_.
```

Comment:
If we were to follow minimalist principles, this would be rewritten as something like. 

Before you can deploy Minion hosts, ensure that you have:
* A Salt Master...
* A Content VIew 
Etc

Reply:
Reworded.